src/pages/en/guides/baas/appwriteio.mdx

```diff
@@ -0,0 +1,777 @@
+---
+title: Appwrite & Astro
+description: Add authentication, storage, and realtime data to your Astro application
+layout: ~/layouts/BackendAsAServiceLayout.astro
+service: Appwrite
+stub: false
+---
+import FileTree from '~/components/FileTree.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+
+[Appwrite](https://appwrite.io/) is a self-hosted backend-as-a-service platform that provides developers with all the core APIs required to build any application.
+
+## Integrating with Astro
+
+In this section, we'll use the [Appwrite JavaScript SDK](https://appwrite.io/docs/getting-started-for-web#getSDK) to connect your Appwrite project to Astro.
+
+### Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+2. **An Appwrite instance, or hosted instance**. If you don't have an account, you can [sign up](https://cloud.appwrite.io/login) for a free account and create a new Appwrite project. You can also use an existing project if you have one. 
+
+3. **Appwrite credentials** - You can find the following credentials in your Appwrite console **Project Overview > API key**. 
+
+    - **Project ID** - The ID of your Project (not the name). 
+
+### Setting up credentials
```

Comment:
The first thing we see in the Appwrite creation flow is this:
<img width="748" alt="image" src="https://user-images.githubusercontent.com/4033662/232090108-80fdbdb7-826e-419c-a7d9-7b5b18943a03.png">
The user will get to this step and expect a parallel step in this guide. Instead, we skip to env variables, but don't see that in the creation flow.

Reply:
Perhaps we should just say skip this flow, as it doesn't always appear if they have another project.